assets/Gruntfile.js

```diff
@@ -386,15 +386,24 @@ module.exports = function (grunt) {
         src: 'views/**/*.html',
         dest: '.tmp/scripts/templates.js',
         options: {
-          module: 'openshiftConsole',
+          module: 'openshiftConsoleTemplates',
+          standalone: true,
           htmlmin: '<%= htmlmin.dist.options %>',
-          usemin: 'scripts/scripts.js'
+          usemin: 'scripts/templates.js'
```

Comment:
Actually yeah we don't need usemin anymore.  Since there will always be a file called scripts/templates.js we can just harcode it into index.html like we do with config.js